<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Elevator</class>
 <widget class="QWidget" name="Elevator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>680</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	border: 1px solid black;
	border-radius: 4px;
	background-color: #ffffff;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="elevatorGroupBox">
     <property name="title">
      <string>ELEVATOR 1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QPushButton" name="button_17">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>130</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>17</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_9">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>160</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_20">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>40</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>20</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_3">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>130</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_6">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_8">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>190</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_1">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>190</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_7">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>220</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QLabel" name="weightLabel">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>100</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0 Kg</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="button_12">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>70</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>12</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_19">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>70</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>19</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_11">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>100</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>11</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>190</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;|&gt;</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="button_18">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>100</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>18</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_14">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>220</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>14</string>
      </property>
     </widget>
     <widget class="QLabel" name="sizeLabel">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>130</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0 Cm2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="button_16">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>160</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>16</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_10">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>130</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>160</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_5">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>70</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_13">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>40</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>13</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_0">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>220</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>G</string>
      </property>
     </widget>
     <widget class="QLabel" name="runLabel">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>60</y>
        <width>40</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <stylestrategy>PreferAntialias</stylestrategy>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>P</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="button_15">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>190</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>15</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_4">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>100</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="closeButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;|&lt;</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="floorGroupBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>38</y>
        <width>120</width>
        <height>202</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Floor 17</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
